core/src/main/scala/kafka/server/ConfigAdminManager.scala

```diff
@@ -154,26 +154,25 @@ class ConfigAdminManager(nodeId: Int,
               throw new InvalidRequestException(s"Unknown resource type ${resource.resourceType().toInt}")
           }
         } catch {
-          case t: Throwable => {
+          case t: Throwable =>
             val err = ApiError.fromThrowable(t)
-            info(s"Error preprocessing incrementalAlterConfigs request on ${configResource}", t)
+            info(s"Error preprocessing incrementalAlterConfigs request on $configResource", t)
             results.put(resource, err)
-          }
         }
       }
     })
     results
   }
 
-  def validateBrokerConfigChange(
+  private def validateBrokerConfigChange(
     resource: IAlterConfigsResource,
     configResource: ConfigResource
   ): Unit = {
     val perBrokerConfig = !configResource.name().isEmpty
     val persistentProps = configRepository.config(configResource)
     val configProps = conf.dynamicConfig.fromPersistentProps(persistentProps, perBrokerConfig)
     val alterConfigOps = resource.configs().asScala.map {
-      case config =>
+      config =>
```

Comment:
Previous code was the standard in really old version of Scala, since then anonymous function style is preferred.